x-pack/plugins/translations/translations/zh-CN.json

```diff
@@ -2852,7 +2852,6 @@
     "timelion.vis.intervalLabel": "时间间隔",
     "uiActions.actionPanel.title": "选项",
     "uiActions.errors.incompatibleAction": "操作不兼容",
-    "visualizations.defaultFeedbackMessage": "想反馈？请在“{link}中创建问题。",
```

Comment:
i18n removed, but not added?

Reply:
thats what our i18n tool does and should be ok